### PR TITLE
Feat:created the implemented the payment page  Team-starlight

### PR DESCRIPTION
**Pull request by jerahmeel200**
# Description

**Closes #issue_number_here**

# Changes proposed

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title and description of the PR is clear and explains the approach.
- [ ] I am making a pull request against the **dev branch** (left side).
- [ ] My commit messages styles matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit test.
- [ ] I am only making changes to files I was requested to.

# Screenshots/Videos

### CODE REVIEW

**Review by jerahmeel200 — COMMENTED** [review]
made some changes

**Review by jerahmeel200 — COMMENTED** [review]
made some changes

**Comment by jerahmeel200** [comment]
phoenix abeg run this merge

**Review by prudentbird — CHANGES_REQUESTED** [review]
Why are you putting page content in a component??

**Comment by jerahmeel200** [comment]
but it is a component foe that particular  page

File: git
Comment (prudentbird) [inline git:1]: What file is this 

Kindly remove

**Review by prudentbird — CHANGES_REQUESTED** [review]

File: git
Comment (jerahmeel200) [reply git:1]: what file?

Comment (prudentbird) [reply git:1]: It's a folder or whatever it is

**Pull request closed without merging**